lib/src/git.rs

```diff
@@ -68,11 +68,32 @@ pub fn import_refs(
     git_repo: &git2::Repository,
     git_settings: &GitSettings,
 ) -> Result<(), GitImportError> {
+    import_some_refs(mut_repo, git_repo, git_settings, |_| true)
+}
+
+/// Reflect changes made in the underlying Git repo in the Jujutsu repo.
+/// If `git_ref_filter` is defined, only branches whose git full reference name
+/// pass the filter will be considered for addition, update, or deletion.
+pub fn import_some_refs<F>(
+    mut_repo: &mut MutableRepo,
+    git_repo: &git2::Repository,
+    git_settings: &GitSettings,
+    git_ref_filter: F,
+) -> Result<(), GitImportError>
+where
+    F: Fn(&str) -> bool,
```

Comment:
Since `F` is short and used in only one place, I would have used `git_ref_filter: impl Fn(&str) -> bool` instead. What do you think?

Reply:
Sure, I also prefer this but was not sure this was an accepted style here.